stanford_sites_helper.module

```diff
@@ -286,3 +286,17 @@ function stanford_sites_helper_stanford_sites_hosted() {
     return FALSE;
   }
 }
+
+/**
+ * Check to see if we are hosted on Acquia.
+ */
+function stanford_sites_helper_is_acquia_hosted() {
```

Comment:
This whole function block could be simplified to 
```
return (getenv('AH_SITE_ENVIRONMENT')) ? TRUE : FALSE;
```

Reply:
_noooooooo_

Reply:
Ok, how about just:
`return getenv('AH_SITE_ENVIRONMENT');`

Reply:
`getenv` already returns false if the environment var does not exist.